meinberlin/apps/votes/forms.py

```diff
@@ -4,9 +4,29 @@
 from meinberlin.apps.votes.models import VotingToken
 
 
-class TokenForm(forms.Form):
+class VotingTokenWidget(forms.MultiWidget):
+
+    def __init__(self, attrs=None):
+        widgets = (
+            forms.TextInput(attrs={'size': '4', 'maxlength': '4'}),
```

Comment:
While this limits the characters actually used by the field, the styling still looks aweful and this contradcits the max_length=40 defined in line 29